### DISCUSSION TITLE
Prepare for introduction of terraform-provider-pipecd

### DISCUSSION THREAD

**Discussion by kentakozuka** [pipe-cd/pipecd#4471] [General]
- [x] Create namespace(pipe-cd)
- [ ] Setup CI
- [ ] Create a repository
- [ ] Preparing and Adding a Signing Key
- [x] Set GPG key and pass phrase to Secrets
- [ ] Publishing Your Provider
- [ ] Sync on Terraform Registry

ref: https://developer.hashicorp.com/terraform/registry/providers/publishing#publishing-to-the-registry

**Comment by t-kikuc** [comment]
I think this is closed.
If not, please open again.
